ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -738,6 +738,44 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure Security Auditing Flags For User-Attributable Events Are Configured Per Local Organizational Requirements
+  platforms: macOS
+  platform: darwin
+  description: |
+    Auditing is the capture and maintenance of information about security-related events. Auditable events often depend on differing organizational requirements.
+  resolution: |
+    Automated method:
+    Ask your system administrator to deploy an MDM profile that disables Bonjour advertising service.
+    Terminal Method: 
+      Perform the following to set the required Security Auditing Flags:
+      Edit the /etc/security/audit_control file and add -fm, ad, -ex, aa, -fr, lo, and -fw to flags. You can also substitute -all for -fm, -ex, -fr, and -fw.
```

Comment:
Says `You can also substitute -all for -fm, -ex, -fr, and -fw.`.

Does this mean the query below has to check if users set `-all` instead of the four settings `-fm, -ex, -fr, -fw`?

Reply:
Fixed. Please review